core/src/main/java/hudson/util/MultipartFormDataParser.java

```diff
@@ -71,4 +72,30 @@ public void cleanUp() {
         for (FileItem item : byName.values())
             item.delete();
     }
+
+    /**
+     * Checks a Content-Type string to assert if it is "multipart/form-data".
+     *
+     * @param contentType Content-Type string.
+     * @return {@code true} if the content type is "multipart/form-data", otherwise {@code false}.
+     * @since TODO
+     */
+    public static boolean isMultiPartForm(@CheckForNull String contentType) {
+        if (contentType == null) {
+            return false;
+        }
+
+        String[] parts = contentType.split(";");
+        if (parts.length == 0) {
+            return false;
+        }
+
+        for (int i = 0; i < parts.length; i++) {
```

Comment:
The tests only test for the first part. Does it even make sense to look at the later parts?